mobify/platform-scaffold
# [Integration Manager] Misc bug fixes

Several small bug fixes I came across preparing for the Show & Tell Demo on Mar 17, 2017/

 **JIRA**: **[WEB-1193](https://mobify.atlassian.net/browse/WEB-1193)** - Show & Tell Goal
 **Linked PRs**: https://github.com/mobify/platform-scaffold/pull/392

## Changes
- Exclude generated file `loader-routes.js` from git
- Make `parseProductHit` more forgiving when price is missing or represented by multiple currencies (stored in `prices` array)
- Tweak routing (remove trailing dash (`-`)) on Demandware routes so that we can navigate from Home directly to a PLP for the top-level categories
- Add Demandware category images (thanks @mitchquik !!) 

## How to test-drive this PR
- Run `npm run dev`
- Preview the Demandware [site](https://preview.mobify.com/?url=http%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2Fdefault%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Load a product category (Electronics is good as it has multiple pricing and missing pricing)
    - Verify all prices show, some may be *N/A*
    - Verify category images are all there (none missing)
- Navigate to a few PDP's (
